docs/google/calibration.md

```diff
@@ -0,0 +1,217 @@
+# Calibration Metrics
+
+Processors that are quantum computers periodically undergo calibrations to
+maintain the quality of the programs that can be run on these processors.
```

Comment:
nit: use pronouns to avoid repetition: s/on these processors/on them/

Reply:
Done.